plugin/trino-base-jdbc/src/main/java/io/trino/plugin/jdbc/BaseJdbcClient.java

```diff
@@ -844,16 +1013,25 @@ public void rollbackCreateTable(ConnectorSession session, JdbcOutputTableHandle
         }
     }
 
+    @Override
+    public boolean supportsFaultTolerantExecution(ConnectorSession session)
```

Comment:
given internal uses of this method it should rather be "shouldUseFaultTolerantExecution"